promscale/installation/promscale-with-timescale-cloud/docker.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Install Promscale using a Docker image
+excerpt: Install Promscale using Docker
+product: promscale
+keywords: [analytics, Docker]
+tags: [install]
+related_pages:
+  - /promscale/:currentVersion:/guides/resource-recomm/
+  - /promscale/:currentVersion:/send-data/
+---
+
+import PromscaleSendData from "versionContent/_partials/_promscale-send-data.mdx";
+
+# Install Promscale Connector using a Docker image with Timescale Cloud
+
```

Comment:
Needs a sentence in here before we leap into the admonition. I would err on the side of being at least slightly repetitive of the admonition, personally. Something like this:

> You can install Promscale Connector from a Docker container using Timescale Cloud. This is useful for developing a proof of concept, or for testing purposes. Do not use this method in a production environment.

Reply:
done!!!